torch/linalg/__init__.py

```diff
@@ -1032,7 +1032,7 @@
   - `'gelsd'` (tridiagonal reduction and SVD)
   - But if you run into memory issues: `'gelss'` (full SVD).
 
-For CUDA input, the only valid driver is `'gels'`, which assumes that :attr:`A` is full-rank.
+For CUDA inputs, two drivers are available: 'gels' and 'gelss'. The 'gels' driver assumes that :attr:A is full-rank. For rank-deficient matrices :attr:A, use the 'gelss' driver.
```

Comment:
```suggestion
For CUDA inputs, two drivers are available: 'gels' and 'gelss'.
```
the drivers are already discussed in the previous paragraph.